test/test_transforms_v2.py

```diff
@@ -647,6 +638,183 @@ def affine_rotated_bounding_boxes(bounding_boxes):
     )
 
 
+def reference_perspective_bounding_boxes(bounding_boxes, *, coefficients, new_canvas_size=None, clamp=True):
```

Comment:
Nit: keep this implementation as well as the one for `reference_perspective_rotated_bounding_boxes` within the `TestPerspective` class, since it seems they are only needed there. It might make the diff smaller too!